mysql-public-db.tf

```diff
@@ -0,0 +1,86 @@
+# Reference to the PostgreSQL/MySql dedicated network external resources
+# ALREADY LOADED IN postgres-public-db.tf
+# data "azurerm_virtual_network" "public_db"
```

Comment:
WDYT moving the data.azurerm_virtual_network.public_db definition from postgres-public-db.tf to vnets.tf ?